### ISSUE TITLE
vlab 0.2.1 show portmap caused traceback

### ISSUE THREAD

**Issue by JohnVo620** [willnx/vlab#1]
```
C:\Windows\System32>vlab --version
vlab, version 0.2.1

C:\Windows\System32>vlab create portmap -a 192.168.1.2 -p ssh -n jv-8007-1
Collecting information about your inventory
Creating a port mapping rule to jv-8007-1 for ssh
OK! Use 'vlab connect onefs --protocol ssh' to access that machine

C:\Windows\System32>vlab show portmap
Looking up port mapping rules
Traceback (most recent call last):
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 14, in run
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\cx_Freeze\initscripts\Console.py", line 26, in run
  File "vlab", line 10, in <module>
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\click-6.7-py3.6.egg\click\core.py", line 722, in __call__
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\click-6.7-py3.6.egg\click\core.py", line 697, in main
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\click-6.7-py3.6.egg\click\core.py", line 1066, in invoke
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\click-6.7-py3.6.egg\click\core.py", line 1066, in invoke
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\click-6.7-py3.6.egg\click\core.py", line 895, in invoke
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\click-6.7-py3.6.egg\click\core.py", line 535, in invoke
  File "C:\Users\willhn\AppData\Local\Programs\Python\Python36\lib\site-packages\click-6.7-py3.6.egg\click\decorators.py", line 17, in new_func
  File "C:\Users\willhn\source\vlab_cli\vlab_cli\subcommands\show\portmap.py", line 22, in portmap
  File "C:\Users\willhn\source\vlab_cli\vlab_cli\lib\portmap_helpers.py", line 122, in port_to_protocol
  File "C:\Users\willhn\source\vlab_cli\vlab_cli\lib\portmap_helpers.py", line 148, in https_to_port
KeyError: 'centos'
```

**Comment by willnx** [comment]
This issue is fixed in `0.2.2` of the vLab CLI; I'll push an updated MSI within the hour.

**Comment by willnx** [comment]
Fix is live, closing issue.
https://vlab-dev.igs.corp/_downloads/vlab-cli-0.2.2-amd64.msi
